eth/api_backend.go

```diff
@@ -290,6 +291,11 @@ func (b *EthAPIBackend) SendTx(ctx context.Context, tx *types.Transaction) error
 		if err := b.eth.seqRPCService.CallContext(ctx, nil, "eth_sendRawTransaction", hexutil.Encode(data)); err != nil {
 			return err
 		}
+		// Retain tx in local tx pool after forwarding, for local RPC usage.
```

Comment:
Doesn't this conflict with tx pool privacy? I mean I think this is the right choice.